front/src/services/lck-utils/columns.ts

```diff
@@ -214,8 +214,7 @@ export function getOriginalColumn (column: LckTableColumn): LckTableColumn {
 }
 
 export function getColumnClass (column: LckTableViewColumn): string {
-  const columnClass = getColumnTypeId(column)
-  switch (columnClass) {
+  switch (column.column_type_id) {
```

Comment:
If you remove the call to `getColumnTypeId`, we won't be able to know the "original" type of a lookup column / vlookup.
Is this what you really want to do ?
And I think the icon displayed in the column header will be empty.

Reply:
For more understanding

Actually with have this: 
![withOrigin](https://user-images.githubusercontent.com/27078725/183960751-6c1dd412-dd48-4cbb-a43b-ea2f81b28194.png)

With this MR will we have:
![withoutOrigin](https://user-images.githubusercontent.com/27078725/183961131-a4282f9a-3a00-42e0-86ab-586668586667.png)

The difference is that now we can see that the column is a FORMULA and so understand why it is not modifiable (same for LOOKED_UP_COLUMN)

Reply:
I understand your point, but I'm not sure how to display it, as I find the information useful.
But, well, this seems a good way to inform the user for the editable - or not - state of the field !